src/openrct2-ui/windows/Multiplayer.cpp

```diff
@@ -122,6 +122,13 @@ static constexpr const uint64 window_multiplayer_page_enabled_widgets[] = {
     (1 << WIDX_CLOSE) | (1 << WIDX_TAB1) | (1 << WIDX_TAB2) | (1 << WIDX_TAB3) | (1 << WIDX_TAB4) | (1 << WIDX_LOG_CHAT_CHECKBOX) | (1 << WIDX_LOG_SERVER_ACTIONS_CHECKBOX) | (1 << WIDX_KNOWN_KEYS_ONLY_CHECKBOX),
 };
 
+static constexpr rct_string_id window_multiplayer_page_titles[] = {
```

Comment:
We try to consistently use TitleCase for local constants. The styles are still mixed, so I get it's somewhat confusing what style to use. I usually check this page when unsure: https://github.com/OpenRCT2/OpenRCT2/wiki/Coding-style